src/install.rs

```diff
@@ -285,9 +285,17 @@ pub fn cinstall(ws: &Workspace, packages: &[CPackage]) -> anyhow::Result<()> {
                 ws.gctx()
                     .shell()
                     .status("Installing", "debugging information")?;
-                let destination_path = build_targets
-                    .debug_info_file_name(&install_path_bin, &install_path_lib)
-                    .unwrap();
+
```

Comment:
This should be windows specific, am I wrong?

Reply:
Correct. I've gone ahead and pushed the changes for macOS to address this feature completely.